packages/components/psammead-podcast-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -460,27 +466,16 @@ exports[`Podcast Promo Examples basic example 1`] = `
   }
 }
 
-.emotion-12 {
-  -webkit-clip-path: inset(100%);
-  clip-path: inset(100%);
-  -webkit-clip: rect(1px,1px,1px,1px);
-  clip: rect(1px,1px,1px,1px);
-  height: 1px;
-  overflow: hidden;
-  position: absolute;
-  width: 1px;
-  margin: 0;
-}
-
 <div>
   <section
-    class="emotion-30 emotion-31"
+    aria-labelledby="some-id"
+    class="emotion-28 emotion-29"
     role="region"
   >
     <div
       class="emotion-6 emotion-7"
     >
-      <i
+      <span
```

Comment:
I would prefer to see this inside the h2 tag as it's associated with that content

Reply:
Nice one - done